packages/application-components/src/components/main-pages/internals/main-page.styles.ts

```diff
@@ -15,6 +15,5 @@ export const MainPageContent = styled.div`
   flex: 1;
   flex-basis: 0;
   overflow: auto;
-  // NOTE: do not change to "padding" as this breaks sticky DataTable styles
```

Comment:
Nit: could you keep and move this comment outside of the styles (L13)?

Reply:
Restored here: 82a180e3727b5946ad094cb809a14302e9c106f2